packages/playground/README.txt

```diff
@@ -0,0 +1,57 @@
+=== WordPress Playground ===
+Contributors: antoniosejas, berislavgrgicak, zieladam
+Tags: playground, staging, sandbox
+Requires at least: 6.0
+Tested up to: 6.4
+Stable tag: 0.1.0
+Requires PHP: 7.0
+License: GPLv2
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+Early preview: Enables running a sandbox of your site using WordPress Playground (https://github.com/WordPress/wordpress-playground)
+
+# WordPress Playground
+
+_v0.1.0_
```

Comment:
Let's label this as Changelog and format similarly to the Classic Editor readme http://plugins.svn.wordpress.org/classic-editor/trunk/readme.txt